packages/desktop/src/renderer/containers/hooks.ts

```diff
@@ -68,9 +67,30 @@ export const useCyclingFocus = (
           }
           setCurrentFocus(focusValue)
           break
+        case 'ArrowDown':
```

Comment:
Double check if it doesn't interrupt the channel page behavior. I mean there's a risk one may have problems changing focus inside message input using keyboard's arrows

Reply:
Yes there was a bug, I wrote some workaround to handle the logic correctly, please take a look at this new part of the code.